Add meta descriptions to Lisk-SDK, closes #371 (#387)
LiskHQ/lisk-docs: modules/ROOT/pages/lisk-elements/packages/constants.adoc

= Lisk Elements Constants
Mona Bärenfänger <mona@lightcurve.io>

> :description: Technical reference for the Constants package of Lisk Elements. It copmrises setup instructions and usage examples.

Reviewer: ```suggestion
:description: Technical references regarding the Constants packages of Lisk Elements. This consists of setup instructions and usage examples.
```
